packages/core/src/blocks/CodeBlockContent/defaultSupportedLanguages.ts

```diff
@@ -1,4 +1,4 @@
-import { bundledLanguagesInfo } from "shiki/bundle/web";
+import { bundledLanguagesInfo } from "shiki";
```

Comment:
@areknawo if we're selecting 1 bundle, doesn't it make more sense to only use `"shiki/bundle/web";` to bring down size at least a little?

Reply:
@YousefED Not sure if it'd work. `bundledLanguagesInfo` is used to verify if specific language ID can be loaded. If it was limited to languages from the web bundle, languages outside of it wouldn't work.

Apart from the check, it might also affect bundling. Importing the full package, all languages are included in the bundle and you can customize which one you choose to support (and externalize others from the bundle). If it was limited to the web bundle, I'm not sure if other language would work at all.

Reply:
If we're fine with only supporting languages from the web bundle we can limit to `shiki/bundle/web` right? I think that's ok for now until we get requests for more esoteric languages.

Also, I'm thinking we might want to shift to forcing users to pass in languages explicitly; while that would not disable syntax highlighting by default, I think it's also important to have a cleaner packaging / bundling experience

Reply:
In that case - yes.

As for the latter - I agree that switching to explicit imports would make sense. The only drawback would be more difficult customization and a question of balance between built-in defaults and languages you have to import (in that case, maybe it makes sense to remove the defaults completely?)

Reply:
decided on call: let's keep as is (full bundle), because otherwise we'll miss several languages. In a next iteration, we'll separate syntax highlighting to a separate package + make it configurable for consumers